tests/src/test/scala/system/basic/WskBasicTests.scala

```diff
@@ -185,6 +185,27 @@ class WskBasicTests
         stderr should include regex (s"""Unable to get package '$name': The requested resource does not exist. \\(code \\d+\\)""")
     }
 
+    it should "return a list of alphabetized packages" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            // Declare 3 packages, create them out of alphabetical order
+            val packageName = "activationBasicTesting"
+            for (i <- 1 to 3) {
+                val name = s"$packageName$i"
+                assetHelper.withCleaner(wsk.pkg, name) {
+                    (pkg, name) =>
+                        pkg.create(name)
+                }
+            }
+            val original = wsk.pkg.list(nameSort = Some(true)).stdout
```

Comment:
it may be prudent to wrap these checks with a `retry` (here and elsewhere) since a list operation may take some time to converge.